### PR TITLE
NMS-14991: Add various strategies and TSS info to usage report

### PR DESCRIPTION
**Pull request by amay94**
### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14991

### CODE REVIEW

File: features/datachoices/pom.xml [hunk 1]
```diff
@@ -97,6 +97,21 @@
...
+            <groupId>org.opennms.core.ipc.sink</groupId>
+            <artifactId>org.opennms.core.ipc.sink.common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.opennms.core.ipc.rpc</groupId>
+            <artifactId>org.opennms.core.ipc.rpc.common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
```
Comment (cgorantla) [inline features/datachoices/pom.xml:110]: Do we need to add these dependencies in features as well ?

Comment (amay94) [reply features/datachoices/pom.xml:110]: I haven't had any issues building it as-is

Comment (cgorantla) [reply features/datachoices/pom.xml:110]: Better to include these bundles in datachoices features.

**Comment by amay94** [comment]
Force push removed attempt to retrieve TSS plugins list, as the `TimeseriesStorageManager` is not loaded by default.

**Review by cgorantla — APPROVED** [review]
LGTM!

**Review by aramosvizcarra — APPROVED** [review]
LGTM!

**Pull request merged**